README.md

```diff
@@ -226,6 +226,29 @@ In summary (from https://stackoverflow.com/a/69673312):
 alembic revision --autogenerate -m "<msg>"
 ```
 
+### Pre-commit Hooks
+
+To ensure code quality and consistency, OpenAdapt uses pre-commit hooks. These hooks will be executed automatically before each commit to perform various checks and validations on your codebase.
```

Comment:
Can we also add a pre-commit hook to split long lines like this one onto multiple lines? 😅

Reply:
Unfortunately there is no pre-commit hook that does this from what I researched. @abrichr I was also considering adding [ruff](https://github.com/astral-sh/ruff)(really fast linter) as a pre-commit hook.

Reply:
Added a new issue for this: https://github.com/OpenAdaptAI/OpenAdapt/issues/new?assignees=&labels=enhancement&projects=&template=new_feature.yml